### ISSUE TITLE
Sufficient description is missing for text present in the tips provided

### ISSUE THREAD

**Issue by puja-iaccessible** [adobe/helix-sidekick-extension#255]
**Description**
The text of each tip uses content like "click here to do this" along with an arrow pointing to the relevant menu item. This is not sufficient for a screen reader user because the screen reader does not indicate that there is an arrow present that is pointing to a menu item.

**To Reproduce**
1. Open the URL https://main--franklinsidekickrepo--puja-iaccessible.hlx.page/
2. Press Alt key to go to the chrome menu and press tab to go to the sidekick extension 
3.Notice the text of tips 

**Expected behavior**
The tip text should instead be more descriptive, for example: 'Click on the Edit button of the sidekick extension toolbar to edit this page". This will sufficiently describe to a screen reader user, the information provided in this tip.

**Screenshots**
Attached are the screenshots.
![Tooltip 2](https://user-images.githubusercontent.com/123964084/219019870-bcf5dd08-03fb-4a74-bf24-55973b93d2dc.png)
![Tooltip 3](https://user-images.githubusercontent.com/123964084/219019921-43dde6a9-9eb0-44d1-8174-8db875902da8.png)

**Additional context**
https://www.w3.org/TR/WCAG21/#sensory-characteristics

When the text alone is unclear, sighted users can examine the surrounding context for clues about the text purpose. Assistive technologies can similarly help non-sighted users by the clear description.

**Comment by rofe** [comment]
@puja-iaccessible this is fixed. See here for updated help texts:
https://www.hlx.live/tools/sidekick/help.json?sheet=help-steps
